### DISCUSSION TITLE
List Layout does not show all search results

### DISCUSSION THREAD

**Discussion by TomLIKG** [microsoft-search/pnp-modern-search#3912] [Q&A]
When I use the ‘List’ layout, some search results are no longer displayed. However, they are displayed in the detailed list layout. What could be the reason for this?

I am using the latest version

Detaillist layout
![Screenshot 2024-08-16 115152](https://github.com/user-attachments/assets/534f379f-8972-4b6e-8600-7132fe7ca815)
List layout
![Screenshot 2024-08-16 115205](https://github.com/user-attachments/assets/35e77c47-05c1-493b-b87f-d2bec76bb1e2)

**Comment by kasperbolarsen** [comment]
Well, the query returned 10 items, so I wonder what the settings on "page" one shows. Is paging set to 6 or something like that :-)

You haven't disabled Paging, right ;-) ??

**Reply by TomLIKG** [reply]
I waited a few more days, as Sharepoint is sometimes a bit delayed. The search results are still not displayed correctly.

Paging isn't disabled :/
![image](https://github.com/user-attachments/assets/be2a562b-a0a1-47a1-8892-6da82b477b54)

**Reply by TomLIKG** [reply]
It looks like it can't list the website pages that were created in the website page library via ‘New’ -> ‘Website page’.

Is it known that these pages cannot be displayed correctly? I have not set up any filters.
